tests/bootstrap.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Bootstraps the Beans Tests.
+ *
+ * @package     Beans\BeansFramework\Tests
+ * @since       1.5.0
+ * @link        http://www.getbeans.io
+ * @license     GNU-2.0+
+ */
+
+if ( version_compare( phpversion(), '5.3.0', '<' ) ) {
+	die( 'Beans Unit Tests require PHP 5.3 or higher.' );
+}
+
```

Comment:
Although the codebase is at PHP 5.2, the testing tools are at 5.6.  Brain/Monkey is a good example.  The risk is minimal as we also do integration and full-on testing too (at PHP 5.2).

Reply:
If the testing tools require PHP 5.6, shouldn't the above message reflect that?

Reply:
@GaryJones You're right.  Fixing in the next commit.